testinfra/build/test_securedrop_deb_package.py

```diff
@@ -119,6 +119,20 @@ def test_deb_package_control_fields_homepage(File, Command, deb):
 
 
 @pytest.mark.parametrize("deb", deb_packages)
+def test_deb_package_contains_no_update_dependencies_file(File, Command, deb):
+    """
+    Ensures the update_python_dependencies script is not shipped via the
+    Debian packages.
+    """
+    deb_package = File(deb.format(
+        securedrop_test_vars.securedrop_version))
+    # Using `dpkg-deb` but `lintian --tag package-installs-python-bytecode`
+    # would be cleaner. Will defer to adding lintian tests later.
```

Comment:
It's on all tests that use `dpkg-deb --contents {}.format(deb_package.path)`. Kept it so that this test wouldn't be missed for upgrade if/when that suggested change happens. Maybe it's better to track these as an issue rather than `TODO` comments as suggested in similar cases, but that's out of scope of this PR to change.